src/client.ts

```diff
@@ -2595,6 +2599,85 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
     );
   }
 
+  async queryThreads(options?: {
```

Comment:
Is there a plan to update client.activeChannels[key].state.threads with the latest date with this query?

Reply:
latest date?

Reply:
Good point. If needed we can, didn't feel any need at the moment.